blocks/newslist/newslist.js

```diff
@@ -46,9 +46,13 @@ function getDescription(queryIndexEntry) {
   const matchingParagraph = longdescriptionElements.find((p) => ABSTRACT_REGEX.test(p.innerText));
   const longdescription = matchingParagraph ? matchingParagraph.innerText : '';
   if (queryIndexEntry.description.length > longdescription.length) {
-    return queryIndexEntry.description;
+    return `<p>${queryIndexEntry.description}</p>`;
   }
-  return longdescription;
+  const oBr = matchingParagraph.querySelector('br');
+  if (oBr) {
+    oBr.remove();
```

Comment:
@jjjaspher is there a reason to remove first `br` tag?

Reply:
@ravkiran the reason for that is there's a visible large space between title and description.

**Migrated Page**
![image](https://github.com/hlxsites/accenture-newsroom/assets/142872664/ac676e96-3879-47e4-8def-1cfdca5b7c4c)

**Newsroom Legacy**
![image](https://github.com/hlxsites/accenture-newsroom/assets/142872664/e725890d-e7b5-4189-abc1-e7f768f3d2b3)

or that can be handle by authoring? since it's just pulling from the article page
![image](https://github.com/hlxsites/accenture-newsroom/assets/142872664/23210a58-feef-4305-ba9b-3c39be623071)